dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/security/KeyVault.java

```diff
@@ -132,7 +132,7 @@ public static PrivateKey loadPrivateKey(String privatePart, String algorithm)
 			try {
 				key = (PrivateKey) provider.invoke(null, privatePart, algorithm);
 			} catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
-				logger.warn("externalPrivateProvider failed with reflection error");
+				logger.warn("Could not decode key as " + algorithm);
```

Comment:
Are the changes in this file leftovers of another PR?

Reply:
arg yes :(